### DISCUSSION TITLE
Getting started with MQTT

### DISCUSSION THREAD

**Discussion by jc003** [cyberman54/ESP32-Paxcounter#970] [Q&A] [answered]
For transferring data using MQTT, currently only ethernet is supported. Do I need to find a board that has an ethernet port? If so, what boards can I use? I looked through the hardware examples, but I couldn't seem to find one with ethernet.

**Comment by cyberman54** [comment, marked as answer]
https://www.olimex.com/Products/IoT/ESP32/ESP32-POE/open-source-hardware
